website/docs/d/oracle_db_system_shapes.html.markdown

```diff
@@ -0,0 +1,88 @@
+---
+subcategory: "Oracle"
+layout: "azurerm"
+page_title: "Azure Resource Manager: Data Source: azurerm_oracle_db_system_shapes"
+description: |-
+  Provides the list of Db System Shapes.
+---
+
+# Data Source: azurerm_oracle_db_system_shapes
+
+This data source provides the list of Db System Shapes in Oracle Cloud Infrastructure Database service.
```

Comment:
Please update all `Db` -> `DB` in this doc
```suggestion
This data source provides the list of DB System Shapes in the Oracle Cloud Infrastructure Database service.
```

Reply:
Thank you, done.